camera_system/intersection_to_block.py

```diff
@@ -86,7 +86,7 @@ def generate_command(self) -> str:
 
         # 縦調整動作ありの場合，縦調整をセットする
         if self.__vertical_flag:
-            command_list += "DS,10,70\n"
+            command_list += "DS,12,70\n"
```

Comment:
```suggestion
            command_list += "DS,12,70,20mm直進(縦調整)\n"
```
コマンドにコメントがあるほうが直前の調整が楽かもしれないです